tools/loc/config.js

```diff
@@ -10,7 +10,9 @@
  * governing permissions and limitations under the License.
  */
 /* global */
-const LOC_CONFIG = '/drafts/localization/configs/config.json';
+import { getUrlInfo } from './utils.js';
+
+const LOC_CONFIG = '/drafts/localization/configs/config-v3.json';
```

Comment:
Once this is merged to main branch - I will raise a follow PR to update the config.json and delete the config-v3.json